data/json/items/gun/shot.json

```diff
@@ -460,8 +461,8 @@
     "id": "remington_870_express",
     "copy-from": "remington_870",
     "type": "GUN",
-    "name": { "str": "7-round shotgun" },
-    "description": "This is a shotgun with capacity for 7 rounds of 12 gauge shot.",
+    "name": { "str": "self-defence pump-action shotgun" },
```

Comment:
Again, this name feels like too much detail. I don't want to cut down on what it's saying, but it's just quite long.